### PR TITLE
Serenity updates - November 2020 - Felix's news

### PR DESCRIPTION
**Pull request by NolwennP**
Left to do:
- [x] Add screenshots
- [x] Add doc links
- [x] Add "More" links

### CODE REVIEW

File: content/updates/2020-11/1-Stop-your-imports-and-exports.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+pim_announcement_alt_img: "The new Stop button in the process tracker"
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Stop your imports and exports
```
Comment (LaureBro) [inline content/updates/2020-11/1-Stop-your-imports-and-exports.md:8]:
```suggestion
# Easily stop ongoing jobs
```
It's not only your jobs, it's any user job if you have the permission, and since it's not only imports and exports, I'd rather have "jobs" in the title.

File: content/updates/2020-11/1-Stop-your-imports-and-exports.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+::: meta-data type="New" features="Productivity" available="mid-November" link-to-doc="../articles/TODO.html"
+:::
+
+A brand new `Stop` button is now displayed in your process tracker. It enables you to stop the Excel and CSV imports/exports you launched. Super helpful whenever you launch a job by mistake and don't want to wait for it to execute until the end. 😉
```
Comment (LaureBro) [inline content/updates/2020-11/1-Stop-your-imports-and-exports.md:12]:
```suggestion
A`Stop` button is now displayed close to jobs in your process tracker. It enables you to stop jobs: imports, exports but also ongoing mass actions or rule executions. This new feature is controlled with permissions, so if you want to be able to stop jobs, ask your admin. Super helpful whenever you or someone from your team launches a job and needs to stop it.
```
@felix-couprie is this correct ⬆️? thank you!

File: content/updates/2020-11/1-Stop-your-imports-and-exports.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+![The new `Stop` button in the process tracker](../img/TODO.png)
+
+This button comes with a dedicated permission `Stop an ongoing job`. If you need it, ask your PIM admin to give you the right to do so.
+
+On a side note, some other jobs such as mass actions or rule executions can also be stopped at any time in the process tracker. So neat!
```
Comment (LaureBro) [inline content/updates/2020-11/1-Stop-your-imports-and-exports.md:18]:
```suggestion
```
I added this part in the intro.

File: content/updates/2020-11/1-Stop-your-imports-and-exports.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+::: more
```
Comment (LaureBro) [inline content/updates/2020-11/1-Stop-your-imports-and-exports.md:20]: Maybe we can add links that are not related to imports/exports? Maybe a link related to the setup of permissions, for imports/exports this will be managed in the export profiles, but @felix-couprie how admins can allow PIM users to stop mass actions and rules execution? TY

File: content/updates/2020-11/4-Progress-bar-and-time-left-for-your-ongoing-jobs.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+pim_announcement_alt_img: "The new progress bar in the process tracker"
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Progress bar and time left for your ongoing jobs
```
Comment (LaureBro) [inline content/updates/2020-11/4-Progress-bar-and-time-left-for-your-ongoing-jobs.md:8]:
```suggestion
# Monitor ongoing jobs progress and time left
```

File: content/updates/2020-11/4-Progress-bar-and-time-left-for-your-ongoing-jobs.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+::: meta-data type="New" features="Productivity" available="mid-November" link-to-doc="../articles/TODO.html"
+:::
+
+You are now able to see an estimation of the remaining time and a progress bar for all your ongoing jobs. So don't hesitate to have a look at your process tracker!
```
Comment (LaureBro) [inline content/updates/2020-11/4-Progress-bar-and-time-left-for-your-ongoing-jobs.md:12]:
```suggestion
Whenever you launch a job, a progress bar and an estimation of the remaining time before the job is completed, are now displayed in the process tracker. You can now easily see how long it will take to process a job, to help you better organize your daily tasks. 
```
I should probably wait until Thursday to check the feature live. I don't know where the progress bar, time left will be visible.(process tracker page or job execution detail page?). Also, a user can see the progress bar and time left for ALL jobs or only his/her jobs? cc @felix-couprie 
Félix, you will also have to redo the screenshots that show the process tracker page the 'started', 'completed' and 'failed' - since badges have changed - on this page: https://help.akeneo.com/pim/serenity/articles/monitor-jobs.html#how-to-monitor

File: content/updates/2020-11/1-Stop-your-imports-and-exports.md (hunk 2 above)
Comment (NolwennP) [reply content/updates/2020-11/1-Stop-your-imports-and-exports.md:12]:
```suggestion
A brand new `Stop` button is now displayed in your process tracker. It enables you to stop jobs: imports, exports but also ongoing mass actions or rule executions. Super helpful whenever you launch a job by mistake and don't want to wait for it to execute until the end. 😉
```

File: content/updates/2020-11/1-Stop-your-imports-and-exports.md (hunk 4 above)
Comment (NolwennP) [reply content/updates/2020-11/1-Stop-your-imports-and-exports.md:20]: I'll change the links to refer to jobs instead of just imports/exports.

Regarding the permission, it's in the user roles.
I'll add the link to how to set up user roles.

File: content/updates/2020-11/4-Progress-bar-and-time-left-for-your-ongoing-jobs.md (hunk 6 above)
Comment (NolwennP) [reply content/updates/2020-11/4-Progress-bar-and-time-left-for-your-ongoing-jobs.md:12]: It's in the job execution detail page. So only users who have access to the job can see the progress.

File: content/updates/2020-11/1-Stop-your-imports-and-exports.md (hunk 2 above)
Comment (LaureBro) [reply content/updates/2020-11/1-Stop-your-imports-and-exports.md:12]:
```suggestion
A brand new `Stop` button is now displayed in your process tracker. It enables you to stop jobs: imports, exports but also ongoing mass actions or rule executions. Super helpful whenever you launch a job by mistake (we're all humans after all) and don't want to wait for it to execute until the end. 😉
```

**Review by LaureBro — APPROVED** [review]

**Comment by felix-couprie** [comment]
OK for me! Thanks

**Review by felix-couprie — COMMENTED** [review]
OK!
